schema/blocks/interstitial.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "section_number": {
+        "type": "string"
+      },
+      "title": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "number": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "pattern": "^Interstitial$"
+      },
+      "primary_content": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "$ref": "../common_definitions.json#/id"
+            },
+            "content": {
+              "$ref": "../common_definitions.json#/guidance_content"
+            }
+          }
+        }
+      },
+      "preview_content": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "$ref": "../common_definitions.json#/id"
+          },
+          "content": {
+            "$ref": "../common_definitions.json#/guidance_content"
+          }
+        }
+      },
+      "secondary_content": {
```

Comment:
primary_content, preview_content and secondary_content are specific to introduction blocks.